smart-contract/assembly/contracts/serializable/Chunk.ts

```diff
@@ -0,0 +1,80 @@
+import { Serializable, Args, Result } from '@massalabs/as-types';
+
+export class ChunkPost implements Serializable {
+  constructor(
+    public filePath: string = '',
```

Comment:
```suggestion
    public filePath: string = '',
```

```suggestion
    public filePath: string = '',
```
Why is file path a string, but in chunkGet a UintArray ?

Reply:
In chunkGet with use the hash of the path

Reply:
yeah this i understand, but im just curious  why we pass a string in chunkPost to convert it to a sha256, but in chunkGet the client is supposed to convert it to a sha256 first ? why doesn't this happen in the ```_getFileChunk``` ?

Reply:
We decided to hash the first time we post the file in the the smart contract so we are sure we keep control on the hash choice (sha256) but it's questionable.